migration-tool/cmd/prom-migrator/main.go

```diff
@@ -181,6 +182,34 @@ loop:
 	log.Info("msg", "exiting!")
 }
 
+type headersFlags struct {
+	headers map[string][]string
+}
+
+func (f headersFlags) String() string {
+	var s string
+	for k, vs := range f.headers {
+		s += k + "=" + strings.Join(vs, ",")
+	}
+	return s
+}
+
+func (f headersFlags) Set(value string) error {
+	k, v, _ := strings.Cut(value, ":")
+	if k == "" {
```

Comment:
You probably want to check `v` here, not `k` (b/c k will contain the actual value)

Reply:
Or just check 3rd bool